.github/workflows/test.yml

```diff
@@ -39,5 +43,10 @@ jobs:
         ls dist
     - name: Check if Manifest was complete
       run: |
-        tar -zxvf dist/flake8-black-*.tar.gz
+        if [ "$RUNNER_OS" == "Windows" ]; then
+          tar zxvf dist/flake8-black-*.tar.gz
+        else
+          tar -zxvf dist/flake8-black-*.tar.gz
+        fi
```

Comment:
No need for the if statement? Linux and macOS should accept either ``tar zxvf ...`` or ``tar -zxvf ...``